### PR TITLE
feat: move react-native-quick-sqlite to op-sqlite

### PR DESCRIPTION
**Pull request by khushal87**
## 🎯 Goal

The goal of the PR is to move to `op-sqlite` rather than `react-native-quick-sqlite` for our offline storage as it is much faster and also allows new architecture support.

## 🎨 UI Changes

<details>
<summary>iOS</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

<details>
<summary>Android</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

## ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] PR targets the `develop` branch
- [x] Documentation is updated
- [x] New code is tested in main example apps, including all possible scenarios
  - [x] SampleApp iOS and Android
  - [x] Expo iOS and Android

### CODE REVIEW

File: docusaurus/docs/reactnative/basics/offline-support.mdx [hunk 1]
```diff
@@ -26,12 +26,12 @@ The following features are currently **NOT** implemented. They will be impleme
 
 First and foremost, make sure to follow all steps from [Migrating to v5](../basics/migrating-from-4.x-to-5.x.mdx) guide. To enable offline support, please follow the given steps:
```
Comment (santhoshvai) [inline docusaurus/docs/reactnative/basics/offline-support.mdx:27]:
```suggestion
First and foremost, make sure to follow all steps from [Migrating to v5](../basics/migrating-from-4.x-to-5.x.mdx) guide. To enable offline support, please follow the given steps:
```

this should be changed to v5 to v6. For later PR i think. But mentioning here to not forget.

File: examples/SampleApp/ios/Podfile
Comment (santhoshvai) [inline examples/SampleApp/ios/Podfile:1]: this seems like big change.. did you test push notifications after this?

**Review by santhoshvai — APPROVED** [review]

File: docusaurus/docs/reactnative/basics/offline-support.mdx (hunk 1 above)
Comment (khushal87) [reply docusaurus/docs/reactnative/basics/offline-support.mdx:27]: yes

File: examples/SampleApp/ios/Podfile
Comment (khushal87) [reply examples/SampleApp/ios/Podfile:1]: I have removed the changes and added the suggested way as per the official docs of op-sqlite. Updated the troubleshoot section as well in push notifications v2 docs.

**Comment by vitorcamachoo** [comment]
Can this be merged to we can move to Expo 51?

**Pull request merged**